docs/installation.rst

```diff
@@ -96,6 +96,14 @@ E-MAIL
 
   Defines maximum number of e-mails that are sent with command ``send_emails_batch``.
 
+.. attribute:: PYMESS_EMAIL_BATCH_MAX_NUMBER_OF_SEND_ATTEMPTS
+
+  Defines maximum number of attempts for sending one e-mail message. Default value is ``3``.
+
+.. attribute:: PYMESS_EMAIL_BATCH_MAX_SECONDS_TO_SEND
+
+  Defines maximum number of seconds to try to send an e-mail message that ended in an ``ERROR`` state. Default value is ``60 * 60`` (1 hour).
```

Comment:
Aby defaultní hodnoty těchto nastavení fungovaly správně, musí se volat odesílání minimálně 3x za hodinu?

Reply:
nn ber to jako dvě dodělené hodnoty

PYMESS_EMAIL_BATCH_MAX_NUMBER_OF_SEND_ATTEMPTS - počet pokusů pro odeslání jednoho e-mailu. Pokud se třikát zavolá command a 3x se nepovede odeslat tak končí v error stavu

PYMESS_EMAIL_BATCH_MAX_SECONDS_TO_SEND - maximální čas po který ze zpráva bude odesílat. To znamená, že i když nebudou použity všechny pokusy tak se po jedné hodině e-mail stejně nepošle, jelikož je už moc starý. Důvod proč jsem tohle udělal je, že když nějaký dobrák změny hodnotu PYMESS_EMAIL_BATCH_MAX_NUMBER_OF_SEND_ATTEMPTS na vyšší číslo poslaly by se i e-maily, které jsou třeba rok staré.

Reply:
pomůžeš s lepším překladem, aby to bylo na první přečtení jasné prosím?

Reply:
Já bych to udělal následovně:
- do popisu PYMESS_EMAIL_BATCH_MAX_NUMBER_OF_SEND_ATTEMPTS přidal věty:

"This controls how many times to try to send an email when first try was not successful. All attempts to send the email have to happen within the timeframe defined by the time of the first attempt and time of the first attempt plus PYMESS_EMAIL_BATCH_MAX_SECONDS_TO_SEND, even if that would mean not all attempts would be made."

a do PYMESS_EMAIL_BATCH_MAX_SECONDS_TO_SEND větu:

"This setting is not dependent on PYMESS_EMAIL_BATCH_MAX_NUMBER_OF_SEND_ATTEMPTS setting and prevents resending too old failed emails if PYMESS_EMAIL_BATCH_MAX_NUMBER_OF_SEND_ATTEMPTS value is raised after some time of use.

Reply:
díky použil jsem

Reply:
jasně, tedy aby se využily všechny 3 nastavené pokusy, musí se volat odesílání minimálně 3x za hodinu. Je to tak?

Reply:
@rubickcz Ano, 3x do hodiny.

Reply:
@matllubos Není zač :)